### DISCUSSION TITLE
Is there guidance for an official pronunciation of "tabyl"?

### DISCUSSION THREAD

**Discussion by mattroumaya** [sfirke/janitor#462] [Q&A] [answered]
My colleagues and I greatly benefit from `janitor ` and are incredibly appreciative of the package! 

We have been running some training sessions, and people tend to say something like: "To calculate frequencies we can use the function tabble, or... tay-bull... table? not sure how to say it...". 

I am genuinely curious if there is an intended way to pronounce `tabyl`! ❓

**Comment by sfirke** [comment, marked as answer]
Good question 😁  There's not a definitive answer, I'd be curious to poll people and see how they say it.  I remarked on this a bit in this issue thread: https://github.com/sfirke/janitor/issues/101#issuecomment-333897361

I was looking for a better name than `tabyl`.  I still don't love `tabyl`, I've just never come up with something I like better.  Pros: It's short, unique, and it looks like if you combined "table" with the 'y' from dplyr/tidyr.  Cons: it's unclear how to say it!

I just say it "table" and kind of aggressively collide with the homophone of base R `table()` - I never use `table()` so it works for me to pretend `table()` doesn't exist but I get how that confusion is not ideal.  But I think there's no wrong answer here.

**Reply by mattroumaya** [reply]
Thanks, Sam! I also just say table, and scarcely use the `table` function. A true toe-may-toe/toe-mah-toe situation! 😸
